pages/news.html

```diff
@@ -88,13 +88,14 @@ <h1>NEWS</h1>
-	<p><strong>19/12/2024</strong> - Uyuni 2024.12 is now available with the new Saline container. Please note, we are experiencing issues with the release of uyuni-tools for Ubuntu 24.04 and Debian 12. Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
-        <p><strong>19/11/2024</strong> - We just released a new set of patches on top of Uyuni 2024.10 that provides important security fixes. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/GL2MN2GQUGQDQP54NI5IB5VGPNHRJH4Z/" target="_blank">announcement</a> to get all the details about the fix and instructions to install the patch. <strong>We strongly recommend you update as soon as possible.</strong></p>
-        <p><strong>25/10/2024</strong> - With a poll on <a href="https://github.com/uyuni-project/uyuni/discussions/9302">our GitHub discussions</a>, the Uyuni community agreed to <strong>move the Uyuni Community Hours to the last Thursday of the month</strong>, always at 16.00 CEST/CET. See you on October 31st for the next session! For the agenda and all the details, please, consult our <a href="https://github.com/uyuni-project/uyuni/wiki/Uyuni-Community-Hours">dedicated wiki page!</a></p>
-	    <p><strong>21/10/2024</strong> - Uyuni 2024.10 is now only available as an openSUSE Leap 15.6 based containers product (to be deployed into an openSUSE Leap Micro 5.5), with support for Ubuntu 24.04, compressed POS image templates and new ISO-8601 date format for API endpoints! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
-	    <p><strong>25/09/2024</strong> - A new <a href="https://calendar.opensuse.org/teams/uyuni" target="_blank">calendar</a> is available in order to track all events related to Uyuni, Uyuni Community Hours included. Feel free to add it as Webcal, Google Calendar or iCalendar URL!</p>
-	    <p><strong>30/08/2024</strong> - Be aware! The GitHub user coffincolors left several comments telling people to install stuff. Do not follow those instructions. Read our <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/YCT34A3Y2J2Y5PUGIWEDZZP3L3EDIGE5/" target="_blank">announcement at our mailing list</a> to know how to proceed!</p>
+            <p><strong>17/02/2025</strong> - We are having the <a href="https://github.com/uyuni-project/uyuni/discussions/9729" target="_blank">discussion about the new base OS for Uyuni</a> since February 5th, as announced <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/WCJIIEYEKSLOJMJOMG7PEZ65S4ILFPVX/" target="_blank">via email<a> and <a href="https://x.com/UyuniProject/status/1887155791330771145" target="_blank">X</a>, until tomorrow 18th, end of the day. If you still didn't join, leave your comments and feedback, in preparation for a final poll to be scheduled after the discussion is complete.</p>
```

Comment:
You missed it here as well @raulillo82 :)

```suggestion
            <p><strong>17/02/2025</strong> - We are having the <a href="https://github.com/uyuni-project/uyuni/discussions/9729" target="_blank">discussion about the new base OS for Uyuni</a> since February 5th, as announced <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/WCJIIEYEKSLOJMJOMG7PEZ65S4ILFPVX/" target="_blank">via email</a> and <a href="https://x.com/UyuniProject/status/1887155791330771145" target="_blank">X</a>, until tomorrow 18th, end of the day. If you still didn't join, leave your comments and feedback, in preparation for a final poll to be scheduled after the discussion is complete.</p>
```